ruby-challenges.rb

```diff
@@ -10,6 +10,24 @@
 num3 = 221
 # Expected output: '221 is odd'
 
+# This method is pretty simple.
+# First, I'll define the method by giving it a name of "even_or_odd" that takes in a number. I'll just 'num' as a placeholder.
+# here we do the logic with an 'if statement'
+# and do a little math using the modulo function that returns the remainder of a number.
+# we make a conditional statement to see if the remainder strictly equals to 0.
+# it returns a true or false value, so we make it print a string interpolated output.
+# if the remainder is 0 or 'true' then it prints "#{num} is even"
+# with an else statement, we make it print a string interpolated output: "{num} "
+def even_or_odd(num)    
+    if num % 2 == 0
+    p "#{num} is even" else p "#{num} is odd"
```

Comment:
Logic looks great!  There is also a Ruby built-in method called .even? that will do some of this work for you.